src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -37,14 +37,6 @@ public override void WillRemoveSubview(UIView uiview)
 				return null;
 			}
 
-			if (!result.UserInteractionEnabled)
-			{
-				// If the child also has user interaction disabled (IOW the child is InputTransparent),
-				// then we also want to exclude it from the hit testing.
-
-				return null;
-			}
```

Comment:
I am not sure why I added this, but it was a long time ago and I did an exhaustive set of tests in this PR at the same time I added this code: https://github.com/dotnet/maui/pull/17286

As far as I can see, this does not look like it is needed as this code path never seems to be hit with other controls - even when the WrapperView is being used.

At this point, I am going to have to rely on the set of tests to validate that this change is fine.